checkov/common/images/graph/image_referencer_provider.py

```diff
@@ -30,11 +29,13 @@ def __init__(self, graph_connector: Union[igraph.Graph, networkx.DiGraph],
     def extract_images_from_resources(self) -> list[Image]:
         pass
 
-    def extract_nodes(self) -> networkx.Graph | igraph.Graph:
+    def extract_nodes(self) -> networkx.Graph | igraph.Graph | None:
         if self.graph_framework == 'NETWORKX':
             return self.extract_nodes_networkx()
-        else:
+        elif self.graph_framework == 'IGRAPH':
             return self.extract_nodes_igraph()
+        else:
+            return None
```

Comment:
Do we handle the case where the the returned subgraph is `None`?

Reply:
the default value of the graph framework is 'NETWORKX', so no. change the function and the 'else' will be the default NETWORKX